src/main/webapp/i18n/en/exercise.json

```diff
@@ -253,7 +253,8 @@
                 }
             },
             "isUsedForTutorial": "Tutor needs to review",
-            "action": "Action"
+            "action": "Action",
+            "exerciseAssessmentDashboard": "Exercise Assessment Dashboard"
```

Comment:
This is a bit long in my opinion. Can you just reuse "Assessment Dashboard" here? This button is already inside the context of a single exercise, so it should be clear that you will end on the assessment dashboard of this particular exercise.